komodo/lint.py

```diff
@@ -56,7 +56,7 @@ def lint_release_name(pkgfile):
             _kerr(
                 pkg=pkgfile,
                 err="Invalid release name suffix. "
-                "Must be of the most -pyXX or -pyXX-rhelY",
+                "Must be of the most -pyXX[X] or -pyXX-rhelY",
```

Comment:
I don't follow this wording. Does is mean "must be of the **form**" (I think it probably does?) or "must be **at** the most" (not sure this makes sense).

Reply:
Right, I did not look at the wording, but it does not make sense currently.. :)

Reply:
Changed "most" to "from"